core/matrix/dense.cpp

```diff
@@ -423,6 +441,38 @@ void Dense<ValueType>::move_to(Sellp<ValueType, int64> *result)
 }
 
 
+template <typename ValueType>
+void Dense<ValueType>::convert_to(SparsityCsr<ValueType, int32> *result) const
+{
+    conversion_helper(result, this,
+                      dense::template make_convert_to_sparsity_csr<
+                          decltype(result), const Dense<ValueType> *&>);
+}
+
+
+template <typename ValueType>
+void Dense<ValueType>::move_to(SparsityCsr<ValueType, int32> *result)
+{
+    this->convert_to(result);
+}
+
+
+template <typename ValueType>
+void Dense<ValueType>::convert_to(SparsityCsr<ValueType, int64> *result) const
+{
+    conversion_helper(result, this,
+                      dense::template make_convert_to_sparsity_csr<
+                          decltype(result), const Dense<ValueType> *&>);
+}
+
+
+template <typename ValueType>
+void Dense<ValueType>::move_to(SparsityCsr<ValueType, int64> *result)
+{
+    this->convert_to(result);
+}
```

Comment:
Missing tests for all dense -> sparsitycsr conversions?

Reply:
Yes, you are right. Thanks!